src/components/QRcode.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+<div>
+  <canvas id="canvas"></canvas>
+</div>
+</template>
+<script>
+import QRCode from 'qrcode'
+export default {
+  props: {
```

Comment:
try to add in some more props for customize the QR code, like colors, sizes ,... ?